openhtf/core/phase_branches.py

```diff
@@ -230,14 +245,23 @@ def _phase_failed(self, phase_rec: test_record.PhaseRecord) -> bool:
     """Returns True if the phase_rec failed; ignores ERRORs."""
     return phase_rec.outcome == test_record.PhaseOutcome.FAIL
 
-  def _check_for_action(self,
-                        running_test_state: 'test_state.TestState') -> bool:
+  def _check_for_action(
+    self,
+    running_test_state: 'test_state.TestState',
+    subtest_rec: Optional[test_record.SubtestRecord] = None
+  ) -> bool:
     """Returns True when the specific set of phases fail."""
     phase_records = running_test_state.test_record.phases
     if not phase_records:
       raise NoPhasesFoundError('No phases found in the test record.')
     if self.previous_phases_to_check == PreviousPhases.LAST:
       return self._phase_failed(phase_records[-1])
+    elif self.previous_phases_to_check == PreviousPhases.SUBTEST and \
+         subtest_rec is not None:
+      for phase_rec in phase_records:
+        if phase_rec.subtest_name == subtest_rec.name and \
```

Comment:
We prefer not using backslash for line continuation, please use braces instead there and at other lines were it is used.

Reply:
Updated to wrap in parentheses.